tests/tests_fcontext.yml

```diff
@@ -1,17 +1,32 @@
 - name: Check if selinux role sets SELinux fcontext mappings
   hosts: all
   become: true
-
+  vars:
+    __str_before: >-
+      staff_u:object_r:user_home_dir_t:s0-s0:c0.c1023
+    __str1: >-
+      /tmp/test_dir1[^ ]+[ ]+directory[
+      ]+staff_u:object_r:user_home_dir_t:s0-s0:c0.c1023
+    __str2: >-
+      /tmp/test_dir2[^ ]+[ ]+directory[
+      ]+staff_u:object_r:user_home_dir_t:s0-s0:c0.c1023
   tasks:
 
+    - name: include test variables
+      import_tasks: set_selinux_variables.yml
+
+    - name: Ensure no mapping to seuser/selevel context
+      assert:
+        that: not __str_before in selinux_role_fcontext.stdout
```

Comment:
Given that there's `selinux_fcontexts_purge: true` bellow, I don't think it's necessary as all local fcontext modification should be dropped by the *initial changes* task

Reply:
dropped check before initial changes purge